tools/isolationchamber.c

```diff
@@ -5,6 +5,8 @@ inherit RoomPlusCode;
 
 #include <ansi.h>
 
+#define SLOOP_FUNCTIONS "/usr/gon/tools/sphere"
```

Comment:
I think I understand what you were trying to do here, but I would really call this #define just SPHERE. Despite the fact we're only using a few of its functions, it's really the entire sphere API that we're integrating with, so why not be explicit about that so other developers know what's up.

That said, the fact that you called your #define SLOOP_FUNCTIONS points to it maybe being a candidate for [refactoring](https://en.wikipedia.org/wiki/Code_refactoring). We talked about doing this tonight by extracting common methods into a separate program which sphere.c can inherit. That new program might be called something like sloop_functions.c -- though I prefer the term 'lib' for "library" over 'functions'; don't all objects contain functions, and what if I want to use some variables too? 

Note: One thing we didn't talk about tonight is inheritance vs. composition in object-oriented design. What you have here with the call_other() is an example of composition. They write books on when to use one or the other, and even if you'd read 'em they mostly go out the window with the way the EotL mudlib is designed, but it's always good to know why you're using whatever you pick. We can talk more about this later.